microsetta_interface/implementation.py

```diff
@@ -784,22 +783,36 @@ def post_account_details(*, account_id=None, body=None):
     return _refresh_state_and_route_to_sink(account_id)
 
 
+@prerequisite([ACCT_PREREQS_MET])
+def source_duplicate_email_check(*, account_id=None, body=None):
```

Comment:
Can you please rename this function to be more clear about the fact that it's not just an email duplicate check?